src/backend/columnar/sql/columnar--9.5-1--10.0-1.sql

```diff
@@ -61,6 +61,9 @@ CREATE TABLE chunk (
 
```

Comment:
Realized that we actually allow unprivileged user to execute `alter_columnar_table_set` for a table that the user doesn't own, I think I should open a seperate pr for that ?

Reply:
https://github.com/citusdata/citus/pull/4748